include/proto/causal.proto

```diff
@@ -0,0 +1,64 @@
+//  Copyright 2018 U.C. Berkeley RISE Lab
+// 
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+// 
+//      http://www.apache.org/licenses/LICENSE-2.0
+// 
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+syntax = "proto2";
+
+enum ConsistencyLevel {
+  SINGLE = 0;
+  CROSS = 1;
```

Comment:
Are there eventually going to be more than two consistency levels?  If not, this can just be a `bool`, no?

Reply:
yeah..but I guess enum is clearer?

Reply:
Fair enough.